src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -150,7 +157,6 @@ paths:
             **An example: 800:20,20-440,440**
```

Comment:
Minor thing: OpenAPI simply writes `Example:` insyead of `An example`, so we should follow that style.